clang/www/cxx_dr_status.html

```diff
@@ -16694,7 +16694,7 @@ <h2 id="cxxdr">C++ defect report implementation status</h2>
     <td><a href="https://cplusplus.github.io/CWG/issues/2813.html">2813</a></td>
     <td>DR</td>
     <td>Class member access with prvalues</td>
-    <td class="unknown" align="center">Unknown</td>
+    <td class="unreleased" align="center">Clang 20</td>
```

Comment:
Given the lack of activity here, this looks to be in danger of slipping - branch is in about a month (going by the llvm 18 [schedule](https://discourse.llvm.org/t/llvm-18-release-schedule/76175))

Reply:
Yeah, if @MitalAshok can't push that through the finish line, Maybe one of us need to take over